### ISSUE TITLE
[Suggestion] Allow specifying external module names for UMD (rollup output.globals)

### ISSUE THREAD

**Issue by FredyC** [jaredpalmer/tsdx#132]
### Current Behavior

![image](https://user-images.githubusercontent.com/1096340/59087581-5a0aa680-8905-11e9-9ad3-ce7a683b9aa9.png)

### Expected behavior

It does guess the names correct in this case but a nicer output would be great. You may notice that guessed name for the last `react-dom` is not even visible. It could lead to unexpected results if it would guess in the wrong way.

### Suggested solution(s)

I wonder why it's not possible to specify externals in a similar fashion as microbundle does. Since it would be optional, it doesn't make the whole experience of zero configuration any worse.

### Additional context

Used in building the https://github.com/JustFly1984/react-google-maps-api

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | 0.6.1
| TypeScript       | 3.5.1
| Browser          | N/A
| Yarn         | 1.16.0
| Operating System | Windows x64

**Comment by sw-yx** [comment]
this one [comes from rollup which comes from node](https://github.com/rollup/rollup/issues/1051). tbh i'd like to suppress this output altogether if we're not even building UMD.

**Comment by FredyC** [comment]
Well, as I said, why not to get inspired by microbundle? I like the way they tackled it in there with CLI option.

https://github.com/developit/microbundle#all-cli-options

**Comment by markerikson** [comment]
I'm playing around with trying to use TSDX with Redux Starter Kit, and the guessing behavior is pretty unhelpful.  It's producing this output:

```js
  (global = global || self, factory(global.ReduxStarterKit = {}, global.redux, global.createNextState, global.selectorator, global.reduxDevtoolsExtension, global.thunk));
```

Pretty sure none of those match the actual global names from the UMD builds of those respective libs.  (Not that I expect a build tool to actually parse or come up with those correctly.)

**Comment by jaredpalmer** [comment]
Sounds good. —external makes sense

**Comment by agilgur5** [comment]
So `tsdx.config.js` was added in #183 and released in v0.9.

I think configuring Rollup's `output.globals` and `externals` is much easier via that than via CLI, so I'm going to close this out
